bridge_adaptivity/module/static/module/js/work_with_cookie.js

```diff
@@ -0,0 +1,12 @@
+function getCookie(name) {
```

Comment:
Please add more comments. For newly created functions here and elsewhere.

Reply:
done

Reply:
Added